cycledash/templates/runs.html

```diff
@@ -77,9 +78,16 @@
           {%- if run['tumor_bam_uri'] %}<span class="label label-info" title="Has an associated tumor BAM">tumor</span>{% endif -%}
           {%- if run['normal_bam_uri'] %}<span class="label label-info" title="Has an associated normal BAM">normal</span>{% endif -%}
         </td>
+        <td class="comments">
+          <span class="comment-bubble {{ '' if run['num_comments'] > 0 else 'no-comment-dimmed'}}">
```

Comment:
Word. Yeah I'd lean towards not doing the explicit check, but it really depends on the day. I believe taking advantage of `0` being falsey is Pythonic, but I'm not a huge fan of that explanation for justifying how I write code…